Update size threshhold

JacobSzwejbka opened the pull request:
Summary: These are the current actual threshholds

Differential Revision: D73056265

JacobSzwejbka commented:
Do these numbers look like what you expect after your size work @swolchok ?

lucylq reviewed (approved):
thanks for updating these!

swolchok commented:
> Do these numbers look like what you expect after your size work @swolchok ?

no particular expectations.

> Im seeing a pretty huge regression from https://github.com/pytorch/executorch/actions/runs/14476679027/job/40603913130?pr=10144
> 
> and I have no idea why.

I recommend using bloaty.

JacobSzwejbka committed:
Update size threshhold (#10214)

Summary:

These are the current actual threshholds

Reviewed By: lucylq

Differential Revision: D73056265
